File: api.v2/python/get_request.py
#curl -H "Content-Type: application/json" \
#     -u user@pt.com:dd2d9ddee2e9cd4861b1f0353375de1b4444d49  \
#    https://api.practitest.com/api/v2/projects.json

import httplib
import requests
from requests.auth import AuthBase
res = requests.get('https://api.practitest.com/api/v2/projects.json', auth=('user@pt.com', 'dd2d9ddee2e9cd4861b1f0353375de1b4444d49'))
print res.status_code
print res.text

File: api.v2/ruby/get_request.rb
#curl -H "Content-Type: application/json" \
# -u my@mail.address:xxx  \
# https://api.practitest.com/api/v2/projects.json

require 'net/http'
require 'net/https'
require 'uri'
require 'json'

URL   = "https://api.practitest.com"
TOKEN = "xxx"
DEVELOPER_EMAIL= "my@mail.address"

uri = URI.parse("#{URL}/api/v2/projects.json")
http = Net::HTTP.new(uri.host, uri.port)
http.use_ssl = true
req = Net::HTTP::Get.new(uri.path)
req.basic_auth DEVELOPER_EMAIL, TOKEN
res = http.request(req)
puts res.body
